src/components/dialogs/export-dialog.js

```diff
@@ -67,64 +175,67 @@ const ExportDialog = ({
         setExportStudyErr('');
         setSelectedFormat('');
         setLoading(false);
-        setDownloadUrl('');
         onClose();
     };
 
-    const handleChange = (event) => {
+    const handleFormatSelectionChange = (event) => {
         let selected = event.target.value;
         setSelectedFormat(selected);
-        setDownloadUrl(getExportUrl(studyUuid, nodeUuid, selected));
     };
 
     const intl = useIntl();
```

Comment:
rules of hooks, move it to the begining of the component

Reply:
OK